keyboards/1upkeyboards/super16/readme.md

```diff
@@ -2,12 +2,15 @@
 
 A 4x4 macropad with RGB underglow.
 
-Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)  
-Hardware Supported: Super 16 Macropad  
+Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)
```

Comment:
Please revert the changes in this file, the deletion of spaces distorts the markdown rendering.